src/translations/enTranslations.json

```diff
@@ -1302,13 +1302,13 @@
           "LiteraryArtsWeb": "Web publication",
           "MentionInPublication": "Publication/mention",
           "MusicScore": "Score/manuscript",
-          "OtherPerformance": "Other performance",
+          "OtherPerformance": "Other",
           "OtherRelease": "Other announcement",
```

Comment:
denne også, eller?
```suggestion
          "OtherRelease": "Other",
```

Reply:
eller dette er en annen slags type kanskje?

Reply:
Dette er for knappene, så tror se skal være som de er :)